middleware/logging/logging.go

```diff
@@ -72,6 +77,7 @@ func Client(logger log.Logger) middleware.Middleware {
 				"kind", "client",
 				"component", kind,
 				"operation", operation,
+				"addr", addr,
```

Comment:
maybe use node

Reply:
node is an interface with no args. how to output all args?

Reply:
> maybe use node

node is an interface with no args. how to output all args?